Change white_listed_* to exclude_* (#73)
urlstechie/urlchecker-action: README.md

| `retry_count`               | <span style="color:green"> optional </span>  | If a request fails, retry this number of times. Defaults to 1    |
| `save`                      | <span style="color:green"> optional </span>  | A path to a csv file to save results to                          |
| `timeout`                   | <span style="color:green"> optional </span>  | The timeout to provide to requests to wait for a response.       |

> | `exclude_urls`              | <span style="color:green"> optional </span>  | A comma separated links to exclude during URL checks.            |

Author: ```suggestion
| `exclude_urls`              | <span style="color:green"> optional </span>  | A comma separated list of links.            |
```

> | `exclude_patterns`          | <span style="color:green"> optional </span>  | A comma separated patterns to exclude during URL checks.         |

Author: ```suggestion
| `exclude_patterns`          | <span style="color:green"> optional </span>  | A comma separated list of patterns.         |
```

> | `exclude_files`             | <span style="color:green"> optional </span>  | Full paths to files to exclude (comma separated list).           |

Reviewer: maybe: 
- A comma separated list of links
- A comma separated list of patterns
or 
- Links to exclude (comma separated)
- Patterns to exclude (comma separated)

Author: Sure! In the future you can just make suggestions for each line that I can directly commit.

| Repository                                                                                          | Workflow (with permalink to YAML) | Example runs |
|-----------------------------------------------------------------------------------------------------|-----------------------------------|-------------|

> | [awesome-rseng](https://github.com/rseng/awesome-rseng)  | [Check URLs in PRs, excludes docs](https://github.com/rseng/awesome-rseng/blob/5f5cb78f8392cf10aec2f3952b305ae9611029c2/.github/workflows/urlchecker.yml)                                   | [Logs](https://github.com/rseng/awesome-rseng/actions?query=workflow%3AURLChecker) |

Reviewer: check  .... exclude
or 
checks ... exclude

> | [awesome-rseng](https://github.com/rseng/awesome-rseng)  | [Check URLs in PRs, excludes docs](https://github.com/rseng/awesome-rseng/blob/5f5cb78f8392cf10aec2f3952b305ae9611029c2/.github/workflows/urlchecker.yml)                                   | [Logs](https://github.com/rseng/awesome-rseng/actions?query=workflow%3AURLChecker) |

Author: ```suggestion
| [awesome-rseng](https://github.com/rseng/awesome-rseng)  | [Check URLs in PRs, exclude docs](https://github.com/rseng/awesome-rseng/blob/5f5cb78f8392cf10aec2f3952b305ae9611029c2/.github/workflows/urlchecker.yml)                                   | [Logs](https://github.com/rseng/awesome-rseng/actions?query=workflow%3AURLChecker) |
```

| [buildtest](https://github.com/buildtesters/buildtest) |  [Check URLs in all commits](https://github.com/buildtesters/buildtest/blob/v0.9.1/.github/workflows/urlchecker.yml)  |  [Logs](https://github.com/HPC-buildtest/buildtest-framework/actions?query=workflow%3A%22Check+URLs%22)           |

> | [us-rse](https://github.com/USRSE/usrse.github.io) |  [Check URLs in PRs, excludes some URL patterns](https://github.com/USRSE/usrse.github.io/blob/abcbed5f5703e0d46edb9e8850eea8bb623e3c1c/.github/workflows/urlchecker.yml)                                 |      [Logs](https://github.com/USRSE/usrse.github.io/actions?query=workflow%3A%22Check+URLs%22)       |

Reviewer: check  .... exclude
or 
checks ... exclude

> | [us-rse](https://github.com/USRSE/usrse.github.io) |  [Check URLs in PRs, excludes some URL patterns](https://github.com/USRSE/usrse.github.io/blob/abcbed5f5703e0d46edb9e8850eea8bb623e3c1c/.github/workflows/urlchecker.yml)                                 |      [Logs](https://github.com/USRSE/usrse.github.io/actions?query=workflow%3A%22Check+URLs%22)       |

Author: ```suggestion
| [us-rse](https://github.com/USRSE/usrse.github.io) |  [Check URLs in PRs, exclude some URL patterns](https://github.com/USRSE/usrse.github.io/blob/abcbed5f5703e0d46edb9e8850eea8bb623e3c1c/.github/workflows/urlchecker.yml)                                 |      [Logs](https://github.com/USRSE/usrse.github.io/actions?query=workflow%3A%22Check+URLs%22)       |
```
